internal/rpcproxy/keymanager.go

```diff
@@ -0,0 +1,132 @@
+// Copyright 2025 Canonical.
+
+package rpcproxy
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"fmt"
+	"regexp"
+
+	jujuparams "github.com/juju/juju/rpc/params"
+	"github.com/juju/utils/v3/ssh"
+	gossh "golang.org/x/crypto/ssh"
+
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/jimm/sshkeys"
+	"github.com/canonical/jimm/v3/internal/openfga"
+)
+
+var isFingerprintRegexp = regexp.MustCompile("^[0-9a-f]{2}(:[0-9a-f]{2}){15}$")
+
+// keyManagerFacade is intended to be a temporary struct used to emulate logic
+// that will eventually live in jujuapi. This struct contains all
+// the api layer logic for SSH key management methods that are currently
+// used by the rpcProxy.
+type keyManagerFacade struct {
+	SSHKeyManager
```

Comment:
Why is this embedded field? I often prefer named fields so it's really clear what struct we're calling methods on.

Reply:
Good catch, I'm not sure either, it's not necessary to be embedded, I've changed it.